docs/content/en/docs/getting-started/_index.md

```diff
@@ -1,72 +1,368 @@
+* The pod `k8ssandra-reaper-k8ssandra-schema-xxxxx` runs once and does not persist.
+{{% /alert %}}
+
+Once all the pods are in the `Running` or `Completed` state, you can check the health of your K8ssandra cluster. There must be **no `PENDING` pods**.
+
+To check the health of your K8ssandra cluster:
+
+1. Confirm that the Cassandra operator is `Ready`:
+
+    ```bash
+    kubectl describe CassandraDataCenter dc1 | grep "Cassandra Operator Progress:"
+       Cassandra Operator Progress:  Ready
+    ```
+
+1. Verify the list of available services:
+
+    ```bash
+    kubectl get services
+    NAME                                        TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                                                 AGE
+    cass-operator-metrics                       ClusterIP   10.99.98.218     <none>        8383/TCP,8686/TCP                                       47m
+    k8ssandra-dc1-all-pods-service              ClusterIP   None             <none>        9042/TCP,8080/TCP,9103/TCP                              47m
+    k8ssandra-dc1-service                       ClusterIP   None             <none>        9042/TCP,9142/TCP,8080/TCP,9103/TCP,9160/TCP            47m
+    k8ssandra-dc1-stargate-service              ClusterIP   10.106.70.148    <none>        8080/TCP,8081/TCP,8082/TCP,8084/TCP,8085/TCP,9042/TCP   47m
+    k8ssandra-grafana                           ClusterIP   10.96.120.157    <none>        80/TCP                                                  47m
+    k8ssandra-kube-prometheus-operator          ClusterIP   10.97.21.175     <none>        443/TCP                                                 47m
+    k8ssandra-kube-prometheus-prometheus        ClusterIP   10.111.184.111   <none>        9090/TCP                                                47m
+    k8ssandra-reaper-k8ssandra-reaper-service   ClusterIP   10.104.46.103    <none>        8080/TCP                                                47m
+    k8ssandra-seed-service                      ClusterIP   None             <none>        <none>                                                  47m
+    kubernetes                                  ClusterIP   10.96.0.1        <none>        443/TCP                                                 53m
+    prometheus-operated                         ClusterIP   None             <none>        9090/TCP                                                47m
+    ```
+
+1. Verify the name of the Cassandra datacenter:
+
+    ```bash
+    kubectl get cassandradatacenters
+    NAME   AGE
+    dc1    51m
+    ```
+
+## Retrieve K8ssandra superuser credentials {#superuser}
+
+You'll need the K8ssandra superuser name and password in order to access Cassandra utilities and do things like generate a Stargate access token.
+
+To retrieve K8ssandra superuser credentials:
+
+1. Retrieve the K8ssandra superuser name:
+
+    ```bash
+    kubectl get secret k8ssandra-superuser -o jsonpath="{.data.username}" | base64 --decode | more
```

Comment:
Cleaner: `kubectl get secret k8ssandra-superuser -o jsonpath="{.data.username}" | base64 --decode ; echo`

The `; echo` gives you the newline I think you're trying to get.

Reply:
You are AWESOME! For some reason, in my iterm, it adds a % control character at the end.